app/services/session.js

```diff
@@ -15,3 +36,28 @@ export default class SessionService extends BaseSessionService {
     super.handleInvalidation(logoutUrl);
   }
 }
+
+// This converts a RouteInfo instance to the corresponding url by collecting all
+// the route params and query params and using the router.urlFor method
+// This is a public API reimplementation of the `transition.intent.url` property
+// More info: https://discord.com/channels/480462759797063690/624403666585124895/897832626616926218
```

Comment:
Very very tiny nitpick:
```suggestion
// This converts a RouteInfo instance to the corresponding url by collecting all
// the route params and query params and using the router.urlFor method.
// This is a public API reimplementation of the `transition.intent.url` property.
// More info: https://discord.com/channels/480462759797063690/624403666585124895/897832626616926218
```

This adds a dot at the end of each comment sentence as is done in the [comment snippet above it](https://github.com/lblod/frontend-worship-decisions/pull/24/files#diff-6ecb3f4231606afab43de2fa51235c22910658ef6201897180c61f62099688a4R17-R20).